src/main/java/com/commercetools/sync/producttypes/utils/ProductTypeSyncUtils.java

```diff
@@ -0,0 +1,56 @@
+package com.commercetools.sync.producttypes.utils;
+
+import com.commercetools.sync.producttypes.ProductTypeSyncOptions;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.producttypes.ProductType;
+import io.sphere.sdk.producttypes.ProductTypeDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import static com.commercetools.sync.producttypes.utils.ProductTypeUpdateActionUtils.buildAttributesUpdateActions;
+import static com.commercetools.sync.producttypes.utils.ProductTypeUpdateActionUtils.buildChangeDescriptionAction;
+import static com.commercetools.sync.producttypes.utils.ProductTypeUpdateActionUtils.buildChangeNameAction;
+import static java.util.stream.Collectors.toList;
+
+public final class ProductTypeSyncUtils {
+
+    /**
+     * Compares all the fields (including the attributes see
+     * {@link ProductTypeUpdateActionUtils#buildAttributesUpdateActions}) of a {@link ProductType} and a
+     * {@link ProductTypeDraft}. It returns a {@link List} of {@link UpdateAction}&lt;{@link ProductType}&gt; as a
+     * result. If no update action is needed, for example in case where both the {@link ProductType} and the
+     * {@link ProductTypeDraft} have the same description, an empty {@link List} is returned.
+     *
+     * @param oldProductType the product which should be updated.
+     * @param newProductType the product draft where we get the new data.
```

Comment:
`the product draft where we get the new data.` -> `the {@link ProductTypeDraft} which should be updated.`

Reply:
7a196a6c573acbde9603e1046cb35d06b450626c